src/ASM/ASMbase.h

```diff
@@ -183,6 +183,10 @@ class ASMbase
   //! \brief Returns the global node number for the given node.
   //! \param[in] inod 1-based node index local to current patch
   virtual int getNodeID(size_t inod, bool = false) const;
+  //! \brief Returns local 1-based index of the element with given global number.
+  //! \details If the given node number is not present, 0 is returned.
+  //! \param[in] globalNum Global node number
```

Comment:
Error in doxy: Should be Global _element_ number

Reply:
done